core/src/main/java/nl/nn/adapterframework/receivers/ReceiverBase.java

```diff
@@ -863,38 +857,38 @@ private void moveInProcessToError(String originalMessageId, String correlationId
 			} 
 			txManager.commit(txStatus);
 		} catch (Exception e) {
-			log.error(getLogPrefix()+"Exception moving message with id ["+originalMessageId+"] correlationId ["+correlationId+"] to error sender, original message: [" + Misc.cleanseMessage(message, getHideRegex(), getHideMethod()) + "]", e);
+			log.error(getLogPrefix()+"Exception moving message with id ["+originalMessageId+"] correlationId ["+correlationId+"] to error sender, original message: [" + message.toString() + "]", e);
 			try {
 				if (!txStatus.isCompleted()) {
 					txManager.rollback(txStatus);
 				}
 			} catch (Exception rbe) {
-				log.error(getLogPrefix()+"Exception while rolling back transaction for message  with id ["+originalMessageId+"] correlationId ["+correlationId+"], original message: [" + Misc.cleanseMessage(message, getHideRegex(), getHideMethod()) + "]", rbe);
+				log.error(getLogPrefix()+"Exception while rolling back transaction for message  with id ["+originalMessageId+"] correlationId ["+correlationId+"], original message: [" + message.toString() + "]", rbe);
```

Comment:
hoeft de `Misc.cleanseMessage` niet meer?

Reply:
Inderdaad, dat gebeurt al in de IbisMaskingLayout

Reply:
Dat klopt, die klasse heb ik zelf nog gemaakt... oops, vergeten!